packages/rx_bloc_cli/mason_templates/bricks/lib_auth_matrix/__brick__/lib/lib_router/routes/auth_matrix_routes.dart

```diff
@@ -25,22 +25,24 @@ class FeatureAuthMatrixRoute extends GoRouteData implements RouteDataModel {
   String get routeLocation => location;
 }
 
-@immutable
+@TypedGoRoute<AuthMatrixOtpRoute>(
+  path: RoutesPath.authMatrixOtp,
+)
 class AuthMatrixOtpRoute extends GoRouteData implements RouteDataModel {
   const AuthMatrixOtpRoute(
-    this.endToEndId,
-  );
+      this.transactionId,
```

Comment:
I don't know if `transactionId` is the most appropriate rename of `endToEndId`, since my mind immediately thinks of bank transfers, but as we know, the auth matrix is capable of more than just that.
Perhaps something more generic like `authenticationId`?
@RomanovaPrime thoughts?

Reply:
The `endToEndId` property has been removed. The `transactionId` is used to track authentication request transcations such as POST /api/v1/auth-matrix/{transactionId}. I agree that we can improve the naming. Do we have any other options other the the following ones ?
- trasncationId
- authenticationId

Reply:
I don't think the suggested names are appropriate here. The reason of using the transactionId is because this simulates how the SQL transaction work. There is a thing called start transaction and commit transaction which corresponds to initiate auth matrix and complete auth matrix.

Follow this link for more details: https://learn.microsoft.com/en-us/sql/t-sql/language-elements/begin-transaction-transact-sql?view=sql-server-ver16

Reply:
I learned something new about SQL today!
In that case, I believe we don't need to change it from `transactionId`